imap_processing/swapi/l2/swapi_l2.py

```diff
@@ -159,14 +159,14 @@ def swapi_l2(
 
     To process science data to L2, we need to:
     - convert counts to rates. This is done by dividing the counts by the
-        TIME_PER_BIN time. TIME_PER_BIN is the exposure time per energy bin which is
-        obtained by dividing the time for one complete sweep
-        (12 s, coarse + fine sweep) by the total energy steps (72),
-        i.e., TIME_PER_BIN = 12/72 = 0.167 s. This will be constant.
+        SWAPI_LIVETIME time. SWAPI_LIVETIME is the acquisition time
+        per energy bin which is obtained by dividing the time for one complete
+        sweep (12 s, coarse + fine sweep) by the total energy steps (72),
+        i.e., SWAPI_LIVETIME = 0.145 s. This will be constant.
```

Comment:
This math doesn't add up anymore. Do you need to update the comment to what is truly being divided here?